### PR TITLE
Add log, cmd and config

### PR DESCRIPTION
**Pull request by guilhem**
and remove useless vendor files (`glide vc`)

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+	"github.com/spf13/viper"
+)
+
+var cfgFile string
+
+// RootCmd represents the base command when called without any subcommands
+var RootCmd = &cobra.Command{
+	Use:   "epoxy",
+	Short: "A Memcached Proxy",
+	Long:  `A description which need to be longer.`,
```
Comment (thomas-maurice) [inline cmd/root.go:32]: much pro

File: handlers/consulmemcached/consulmemcached.go [hunk 2]
```diff
@@ -53,27 +54,20 @@ func (h *Handler) Prepend(cmd common.SetRequest) error {
...
 	errorOut := make(chan error)
+	defer close(errorOut)
 
 	fmt.Println(">> Get operation <<")
 
 	for idx, bk := range cmd.Keys {
 		item, err := h.mc.Get(string(bk))
 
 		if err != nil {
+			log.WithFields(log.Fields{
```
Comment (thomas-maurice) [inline handlers/consulmemcached/consulmemcached.go]: log.WithError(err).Debug("blah")

File: main.go [hunk 3]
```diff
@@ -1,27 +1,14 @@
...
-			Type: server.ListenTCP,
-			Port: 11211,
-		},
-		server.Default,
-		orcas.L1Only,
-		consulmemcached.New(mc),
-		handlers.NilHandler,
-	)
+	if err := cmd.RootCmd.Execute(); err != nil {
+		log.WithFields(log.Fields{
```
Comment (thomas-maurice) [inline main.go]: log.WithError

**Review by thomas-maurice — CHANGES_REQUESTED** [review]
some logging to be reworked

File: cmd/root.go (hunk 1 above)
Comment (BarthV) [reply cmd/root.go:32]: wow such quality, very impressed
:dog2:

Comment (thomas-maurice) [reply cmd/root.go:32]: :wolf:

**Pull request merged**